app/views/users/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<h2>Own photos</h2>
+
+<% @user.own_photos.each do |photo| %>
+  <div class="row mb-4">
+    <div class="col-md-6 offset-md-3">
+      <div class="card">
+        <%= image_tag photo.image, class: "card-img-top" %>
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="" alt="">
```

Comment:
The <img> tag inside the comment list item has an empty src and alt attribute. It's good practice to provide meaningful alt text for accessibility and fill in the src attribute with the commenter's avatar or a placeholder image if the commenter doesn't have an avatar.